recipes-benchmark/s-suite/s-suite_git.bb

```diff
@@ -0,0 +1,21 @@
+SUMMARY = "Small collection of benchmarks for storage I/O"
+LICENSE = "GPLv2"
+LIC_FILES_CHKSUM = "file://COPYING;md5=b529aaa6a0c50f15d29f89609b5c22f3"
+
+SRCREV = "79698f645bfb28d0d966484ddad3a1efb562246d"
+SRC_URI = "git://github.com/Algodev-github/S.git;protocol=https;branch=master"
+
+S = "${WORKDIR}/git"
+
+DEPENDS = "rsync"
+RDEPENDS_${PN} = "bash bc coreutils gawk g++ gcc fio libaio libaio-dev sysstat"
+
+do_install() {
+    mkdir -p ${D}/opt/
+    cp -Rd --preserve=mode,timestamps ${S} ${D}/opt/S-suite
```

Comment:
using 'install' command should be preferred to 'cp'.

Reply:
I need to preserve mode and timestamps, I couldn't find that 'install' could preserve the mode, only the timestamps. If so I have to stick with cp.